engine/src/main/java/org/camunda/bpm/engine/impl/el/ExpressionManager.java

```diff
@@ -16,27 +16,11 @@
  */
 package org.camunda.bpm.engine.impl.el;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
-import org.camunda.bpm.engine.delegate.VariableScope;
-import org.camunda.bpm.engine.impl.core.variable.scope.AbstractVariableScope;
-import org.camunda.bpm.engine.impl.javax.el.ArrayELResolver;
-import org.camunda.bpm.engine.impl.javax.el.CompositeELResolver;
-import org.camunda.bpm.engine.impl.javax.el.ELContext;
-import org.camunda.bpm.engine.impl.javax.el.ELResolver;
-import org.camunda.bpm.engine.impl.javax.el.ExpressionFactory;
-import org.camunda.bpm.engine.impl.javax.el.FunctionMapper;
-import org.camunda.bpm.engine.impl.javax.el.ListELResolver;
-import org.camunda.bpm.engine.impl.javax.el.MapELResolver;
-import org.camunda.bpm.engine.impl.javax.el.ValueExpression;
-import org.camunda.bpm.engine.impl.juel.ExpressionFactoryImpl;
-import org.camunda.bpm.engine.test.mock.MockElResolver;
-import org.camunda.bpm.engine.variable.context.VariableContext;
-
-
 /**
+ * <p>
+ * Original ExpressionManager class is renamed to {@link JuelExpressionManager}
+ * to make el pluggable.
+ * </p>
```

Comment:
Such pointers often turn out to be out of date pretty quickly. The Java type hierarchy will quickly point people to the existing implementations of this interface, I believe that is sufficient.
```suggestion
```